lib/pure/asyncmacro.nim

```diff
@@ -440,7 +399,24 @@ macro multisync*(prc: untyped): untyped =
   result.add(asyncSingleProc(asyncPrc))
   result.add(sync)
 
-proc await*[T](x: T) =
-  ## The 'await' keyword is also defined here for technical
-  ## reasons. (Generic symbol lookup prepass.)
-  {.error: "Await only available within .async".}
+# seems simpler to define it here, than to define it in each multisync
```

Comment:
Not sure what this means, the template doesn't just apply to multisync, does it?

Reply:
i think this might be outdated